Sources/PackageRegistry/RegistryClient.swift

```diff
@@ -1024,23 +959,27 @@ extension RegistryClient {
 }
 
 extension RegistryClient {
-    public struct PublishRequirements {
-        public let metadata: Metadata
+    public struct PublishConfiguration {
         public let signing: Signing
 
-        public struct Metadata {
-            public let location: [MetadataLocation]
-        }
-
-        public enum MetadataLocation {
-            case request
-            case archive
+        public init(signing: Signing) {
+            self.signing = signing
         }
 
         public struct Signing {
```

Comment:
This doesn't match the proposal yet does it?

Reply:
@yim-lee not sure? what is missing?

Reply:
The properties/parameters `acceptedSignatureFormats` and `trustedRootCertificates` are either not needed or should be renamed I think.